src/backend/src/services/tasks.services.ts

```diff
@@ -149,12 +160,16 @@ export default class TasksService {
     const originalTask = await prisma.task.findUnique({
       where: { taskId },
       include: {
-        wbsElement: { include: { project: { ...projectQueryArgs } } }
+        wbsElement: { include: { project: { ...projectQueryArgs } } },
+        assignees: true
       }
     });
     if (!originalTask) throw new NotFoundException('Task', taskId);
     if (originalTask.dateDeleted) throw new DeletedException('Task', taskId);
 
+    const originalAssignees = originalTask.assignees.map((user) => user.userId);
+    const newAssignees = assignees.filter((user) => !originalAssignees.includes(user));
```

Comment:
same here for use, also probably good to do it for originalAssignees: originalAssigneeIds, and newAssignees: newAssigneeIds